randovania/games/samus_returns/assets/hash_words.json

```diff
@@ -0,0 +1,114 @@
+[
+  "Samus",
+  "Metroid",
+  "Alpha",
+  "Gamma",
+  "Zeta",
+  "Omega",
+  "Larva",
+  "Queen",
+  "Baby",
+  "Hatchling",
+  "Arachnus",
+  "Diggernaut",
+  "Proteus",
+  "Ridley",
+  "Animworm",
+  "Autoad",
+  "Autrack",
+  "BigBug01",
+  "BigBug02",
+  "Blob",
+  "Thrower",
+  "Chute",
+  "Leech",
+  "Drivel",
+  "Gawron",
+  "Gigadora",
+  "Glow",
+  "Fly",
+  "Gravitt",
+  "Gryncore",
+  "Gullug",
+  "Gunzoo",
+  "Halzyn",
+  "Hornoad",
+  "Meboid",
+  "Moheek",
+  "Moto",
+  "Mumbo",
+  "Paraby",
+  "Ramulken",
+  "Red",
+  "Plants",
+  "Rock",
+  "Icicle",
+  "Taramarga",
+  "Tsumuri",
+  "Wallfire",
+  "Power",
+  "Charge",
+  "Ice",
+  "Wave",
+  "Grapple",
+  "Spazer",
+  "Plasma",
+  "Beam",
+  "Missile",
+  "Super",
+  "Bomb",
+  "Varia",
+  "Gravity",
+  "Suit",
+  "Morph",
+  "Spider",
+  "Spring",
+  "Ball",
+  "High",
+  "Jump",
+  "Boots",
+  "Space",
+  "Screw",
+  "Attack",
+  "Scan",
+  "Pulse",
+  "Lightning",
+  "Armor",
+  "Beam",
+  "Burst",
+  "Phase",
+  "Drift",
+  "Tank",
+  "Energy",
+  "Aeion",
+  "Item",
+  "Sphere",
+  "Recharge",
+  "Station",
+  "Save",
+  "Teleport",
+  "Chozo",
+  "Seal",
+  "Ability",
+  "Artifact",
+  "Big",
+  "Orb",
+  "Gunship",
+  "Door",
+  "Inactive",
+  "Closed",
+  "Green",
+  "Yellow",
+  "Map",
+  "Boost",
+  "Reserve",
+  "amiibo",
```

Comment:
ngl, I think we should just screw the normal stylization and enforce Title Case.

Reply:
Oh is the lowercase gonna cause issues? I'll fix it

Reply:
Probably wont cause issues, it just looks bad :stuck_out_tongue_closed_eyes: